civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6692,14 +6700,17 @@ function _civihr_employee_portal_create_default_reports_configuration() {
  */
 function civihr_employee_portal_get_markup_for_extension($resource = '', $scriptExtensionKey, $markupModuleName) {
   $script = CRM_Core_Config::singleton()->debug ? "{$resource}.js" : "{$resource}.min.js";
+  $script_path_related_to_extension = 'js/dist/' . $script;
```

Comment:
I'm not sure if this is correct. The line above this one indicates that this function should be able to also load non-distributive files, which I think are outside the `js/dist` folder, right? I think we shouldn't make any assumptions about the paths here

Reply:
I remember that, in the past, we used to include the src files instead of the dist one when debugging was enable. I know that Rene did some work to use sourcemaps instead, but I don't know if this works for all the extensions of not. If it's for all of them, then I don't think we the if condition checking if debugging is enabled or not.

Reply:
I was thinking about this as well, I don't think we use src files anymore. Should we proceed and get rid of this condition?

Reply:
Yes, let's do that